includes/Admin/Settings_Page.php

```diff
@@ -0,0 +1,270 @@
+<?php
+/**
+ * Settings page for AI Experiments.
+ *
+ * @package WordPress\AI
+ *
+ * @since 0.1.0
+ */
+
+declare(strict_types=1);
+
+namespace WordPress\AI\Admin;
+
+use WordPress\AI\Experiment_Registry;
+
+/**
+ * Manages the admin settings page for AI experiments.
+ *
+ * @since 0.1.0
+ */
+class Settings_Page {
+
+	/**
+	 * The experiment registry instance.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @var \WordPress\AI\Experiment_Registry
+	 */
+	private Experiment_Registry $registry;
+
+	/**
+	 * The option name for the global experiments toggle.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @var string
+	 */
+	private const GLOBAL_OPTION = 'ai_experiments_enabled';
+
+	/**
+	 * The settings page slug.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @var string
+	 */
+	private const PAGE_SLUG = 'ai-experiments';
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @param \WordPress\AI\Experiment_Registry $registry The experiment registry.
+	 */
+	public function __construct( Experiment_Registry $registry ) {
+		$this->registry = $registry;
+	}
+
+	/**
+	 * Initializes the settings page hooks.
+	 *
+	 * @since 0.1.0
+	 *
+	 * @return void
+	 */
+	public function init(): void {
+		add_action( 'admin_menu', array( $this, 'register_menu' ) );
+		add_action( 'admin_init', array( $this, 'register_settings' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_styles' ) );
```

Comment:
Similar to what I'm saying above, this is not an ideal pattern: I realize you're checking for the specific screen in the callback and bail early if it's not that, but better would be to really only hook in this method on the specific screen: `"load-{$hook_suffix}"` is your friend :)